nautobot/core/api/urls.py

```diff
@@ -15,11 +15,7 @@
     license=openapi.License(name="Apache v2 License"),
 )
 
-schema_view = get_schema_view(
-    openapi_info,
-    validators=["flex", "ssv"],
-    public=True,
-)
+schema_view = get_schema_view(openapi_info, validators=["flex", "ssv"], public=True, permission_classes=())
```

Comment:
@jathanism @glennmatthews @imranh2 Do we recall why `permission_classes` was set to `()` here?